templates/legal/websites.html

```diff
@@ -133,15 +145,6 @@ <h1>Canonical websites</h1>
         <li class="p-list__item">
           <a href="https://voices.canonical.com">voices.canonical.com</a>
```

Comment:
Wasn't touched in this PR, but this link doesn't work.

Reply:
great catch - removed and docs updated